### PR TITLE
Bump v0.7.0+v0.25.0

### PR DESCRIPTION
**Pull request by arg0d**
Create `docs/RELEASE.md` containing the release procedure.

Refactor configuration options from `README.md` into `docs/CONFIGURATION.md`.

### CODE REVIEW

File: docs/RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+upstream `uniffi-rs` version in the build metadata component (denoted as `+`). The upstream explicit
+upstream `uniffi-rs` aids consumer to target the same upstream version when mixing multiple
+generators, e.g. `uniffi-bindgen-cs` and `uniffi-bindgen-go`.
+```
+v0.6.0+v0.25.0
+```
+
+## Update version in [bindgen/Cargo.toml](../bindgen/Cargo.toml)
+
+Note that the version in [bindgen/Cargo.toml](../bindgen/Cargo.toml) does is not prefixed with `v`,
```
Comment (Lipt0nas) [inline docs/RELEASE.md:17]:
```suggestion
Note that the version in [bindgen/Cargo.toml](../bindgen/Cargo.toml) is not prefixed with `v`,
```

**Review by Lipt0nas — APPROVED** [review]
The version table is missing v0.7.0 entry for the uniffi version as well

**Comment by arg0d** [comment]
> The version table is missing v0.7.0 entry for the uniffi version as well

The table only records versions where `uniffi-rs` version was changed. In this case `0.25` was already used for previous version.

**Comment by Lipt0nas** [comment]
> > The version table is missing v0.7.0 entry for the uniffi version as well
> 
> The table only records versions where `uniffi-rs` version was changed. In this case `0.25` was already used for previous version.

Ah, makes sense 👍

**Pull request merged**